docs/standard/class-library-overview.md

```diff
@@ -82,9 +83,9 @@ The [!INCLUDE[dnprdnshort](../../includes/dnprdnshort-md.md)] includes classes,
  For more information on namespaces and type names, see [Common Type System](../../docs/standard/base-types/common-type-system.md).  
   
 ## System Namespace  
- The <xref:System> namespace is the root namespace for fundamental types in the [!INCLUDE[dnprdnshort](../../includes/dnprdnshort-md.md)]. This namespace includes classes that represent the base data types used by all applications: <xref:System.Object> (the root of the inheritance hierarchy), <xref:System.Byte>, <xref:System.Char>, <xref:System.Array>, <xref:System.Int32>, <xref:System.String>, and so on. Many of these types correspond to the primitive data types that your programming language uses. When you write code using .NET Framework types, you can use your language's corresponding keyword when a .NET Framework base data type is expected.  
+ The <xref:System> namespace is the root namespace for fundamental types in .NET. This namespace includes classes that represent the base data types used by all applications: <xref:System.Object> (the root of the inheritance hierarchy), <xref:System.Byte>, <xref:System.Char>, <xref:System.Array>, <xref:System.Int32>, <xref:System.String>, and so on. Many of these types correspond to the primitive data types that your programming language uses. When you write code using .NET Framework types, you can use your language's corresponding keyword when a .NET Framework base data type is expected.  
   
- The following table lists the base types that the [!INCLUDE[dnprdnshort](../../includes/dnprdnshort-md.md)] supplies, briefly describes each type, and indicates the corresponding type in Visual Basic, C#, C++, and JScript.  
+ The following table lists the base types that .NET supplies, briefly describes each type, and indicates the corresponding type in Visual Basic, C#, C++, and JScript.  
```

Comment:
should we pull out JScript info and add F# instead? /cc @cartermp

Reply:
Yep!

Reply:
I agree, @mairaw. @cartertmp, could you review the table after I merge the revision to make sure that I've correctly described the F# data types? I'm unsure about object. (Is there a need for an object in F#?)